hw/xbox/smbus_cx25871.c

```diff
@@ -23,56 +23,54 @@
 #include "hw/i2c/smbus.h"
 #include "smbus.h"
 
-//#define DEBUG
+// #define DEBUG
+#ifdef DEBUG
+# define XBOX_DPRINTF(format, ...)     printf(format, ## __VA_ARGS__)
+#else
+# define XBOX_DPRINTF(format, ...)     do { } while (0)
+#endif
 
 typedef struct SMBusCX25871Device {
     SMBusDevice smbusdev;
-
     uint8_t registers[256];
 } SMBusCX25871Device;
 
 static void cx_quick_cmd(SMBusDevice *dev, uint8_t read)
 {
-#ifdef DEBUG
-    printf("cx_quick_cmd: addr=0x%02x read=%d\n", dev->i2c.address, read);
-#endif
+    XBOX_DPRINTF("cx_quick_cmd: addr=0x%02x read=%d\n", dev->i2c.address, read);
 }
 
 static void cx_send_byte(SMBusDevice *dev, uint8_t val)
 {
-#ifdef DEBUG
-    printf("cx_send_byte: addr=0x%02x val=0x%02x\n",
-           dev->i2c.address, val);
-#endif
+    XBOX_DPRINTF("cx_send_byte: addr=0x%02x val=0x%02x\n",
+                 dev->i2c.address,
+                 val);
```

Comment:
For the debug stuff in particular I'd like to keep it short. so on same line was better in my opinion.